package.json

```diff
@@ -24,6 +24,11 @@
     "test:e2e": "playwright test"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
```

Comment:
Sorry I meant to use MUI icons from the existing react-icons package: https://react-icons.github.io/react-icons/icons?name=md

Reply:
oh okay, let me change it back